commodore/64/lib/stdlib.c

```diff
@@ -0,0 +1,9 @@
+// place holder for stdlib "abort" function.
+
+// Not sure what the best way to handle this is on C64.
+
+void abort()
+{
+    asm volatile("brk" : );
```

Comment:
Looking at the example set by cc65, this should probably actually return to BASIC, at least if the memory map is set up to keep the BASIC environment usable.

The behavior of BRK without a custom monitor or handler is pretty questionable though; an infinite loop is probably safer until we really get into the weeds of how this should work.